modules/ws-client/package.json

```diff
@@ -5,7 +5,8 @@
     "postbuild": "cp -r static/* dist/"
   },
   "devDependencies": {
-    "parcel": "^2.3.2"
+    "parcel": "^2.3.2",
+    "process": "^0.11.10"
```

Comment:
Just wondering why do we need to add this?

Reply:
No idea, that was added automagically when I ran the app. Perhaps @davesmith00000 can help answer your question.

Reply:
Couldn't say for sure, but I think it's just a transitive dependency for Parcel.js. However it's only in the `devDependencies` section, so it won't be bundled in your app.

Reply:
But really at this point, as this project stands, all we're using npm/yarn for is managing parcel. So if parcel was installed globally or via a nix-shell or something, then all the work being done here might be done by sbt or Mill instead.

Now of course if Gabriel wanted to start pulling in other npm libs or npm packaged css or soemthing, then we're back to needing this set up or to using something like scalajs-bundler. But... so close to not needing NPM. So close!

Reply:
ah, I got it, thanks @gvolpe and @davesmith00000 for your explanation!